api/src/main/resources/fireCloud.yaml

```diff
@@ -344,3 +462,104 @@ definitions:
       userEmail:
         type: string
         description: User's email
+
+  Workspace:
+    description: ''
+    required:
+      - namespace
+      - name
+      - workspaceId
+      - authorizationDomain
+      - bucketName
+      - createdDate
+      - lastModified
+      - createdBy
+      - attributes
+      - accessLevels
+      - authDomainACLs
+      - isLocked
+    type: object
+    properties:
+      namespace:
+        type: string
+        description: The namespace the workspace belongs to
+      name:
+        type: string
+        description: The name of the workspace
+      authorizationDomain:
+        type: array
+        items:
+          $ref: '#/definitions/ManagedGroupRef'
+        description: The list of groups to form the Authorization Domain (empty if no Authorization Domain is set)
+      workspaceId:
+        type: string
+        description: A UUID associated with the workspace
+      bucketName:
+        type: string
+        description: The name of the bucket associated with the workspace
+      createdDate:
+        type: string
+        format: date-time
+        description: 'The date the workspace was created in yyyy-MM-ddTHH:mm:ss.SSSZZ format'
+      lastModified:
+        type: string
+        format: date-time
+        description: 'The date the workspace was last modified in yyyy-MM-ddTHH:mm:ss.SSSZZ format'
+      createdBy:
+        type: string
+        description: The user who created the workspace
+      attributes:
+        type: object
+        description: Map[String, Attribute]
+      accessLevels:
+        type: object
+        description: Map[String, RawlsGroupRef]
+      authDomainACLs:
+         type: object
+         description: Map[String, RawlsGroupRef]
+      isLocked:
+        type: boolean
+        description: Can the Workspace currently be modified?
+
+  WorkspaceIngest:
+    required:
+      - namespace
+      - name
+      - attributes
+    type: object
+    properties:
+      namespace:
+        type: string
+        description: New workspace namespace
+      name:
+        type: string
+        description: New workspace name
+      attributes:
+        type: object
+        additionalProperties:
+          type: string
+        description: Map of attributes
+      authorizationDomain:
+        type: array
+        items:
+          $ref: '#/definitions/ManagedGroupRef'
+        description: The list of groups to form the Authorization Domain (empty if no Authorization Domain is set)
+
+  WorkspaceListEntry:
+    required:
+      - workspace
+    properties:
+      # We are ignoring this part.
```

Comment:
Do you mean this is part of the normal Swagger spec that we're omitting? Or there are fields in the expected request body (from the client) we aren't parsing (yet)? Or something else?

Reply:
This particular response entity wasn't defined in the FC API spec. (It doesn't say what response type is returned from api/workspaces.) So I was reverse engineering it from the JSON I saw in the browser.

I found out subsequently that it will return WorkspaceResponse from the Rawls spec:

https://github.com/broadinstitute/rawls/blob/develop/core/src/main/resources/swagger/rawls.yaml#L5034

I have updated the PR to use that instead (with a comment at the top.)